components/admin/permit-holders/permit-holder-information/EditModal.tsx

```diff
@@ -35,60 +36,14 @@ export default function EditUserInformationModal({
 }: EditUserInformationModalProps) {
   const { isOpen, onClose, onOpen } = useDisclosure();
 
-  // Personal information state
-  const [firstName, setFirstName] = useState('');
-  const [middleName, setMiddleName] = useState<string | null>('');
-  const [lastName, setLastName] = useState('');
-  const [dateOfBirth, setDateOfBirth] = useState(formatDate(new Date(), true));
-  const [gender, setGender] = useState<Gender>(applicant.gender);
-
-  // Contact information state
-  const [email, setEmail] = useState<string | null>('');
-  const [phoneNumber, setPhoneNumber] = useState('');
-
-  // Home address information state
-  const [addressLine1, setAddressLine1] = useState('');
-  const [addressLine2, setAddressLine2] = useState('');
-  const [city, setCity] = useState('');
-  const [postalCode, setPostalCode] = useState('');
-
-  //   TODO: Add error states for each field (post-mvp)
-
   const [loading, setLoading] = useState(false);
```

Comment:
Can we change `loading` to be a prop instead? Then we can pass in the loading state from the GraphQL call